From 3327c51ed643f32b3e23318e02023abd20c5880d Mon Sep 17 00:00:00 2001
From: Angela Fong <angela.fong@joyent.com>
Date: Fri, 3 Aug 2018 11:42:17 -0700
Subject: [PATCH] TRITON-664: Firewall rules: update cloudapi and fwapi docs to
 include more examples and mention about fwrule_version Reviewed by: Cody
 Peter Mello <melloc@writev.io> Approved by: Cody Peter Mello
 <melloc@writev.io>

---
 docs/index.md | 30 +++++++++++++++++++++---------
 1 file changed, 21 insertions(+), 9 deletions(-)

diff --git a/docs/index.md b/docs/index.md
index 33acc22..21316f7 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -10,7 +10,7 @@ apisections: Rules, Firewalls
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 # FWAPI (Firewall API)
@@ -124,18 +124,17 @@ There are two more steps necessary:
 * Updating all webserver VMs with tag **www**
 * Updating all database VMs with tag **db**
 
-Please see the [examples](examples.html) page for more examples.
+Please see the [CloudAPI Guide](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md#firewallrules)
+for more examples.
 
 # Rule Syntax
 
-Rules are written in a Doman Specific Language (DSL). References for it are:
-
-* The [rule syntax guide](rules.html) has the complete syntax.
-* The [railroad diagram](railroad-diagram.xhtml) has an easier to read
-  format.
-* The [cloudapi firewall rules documentation](http://wiki.joyent.com/wiki/display/jpc2/Firewall+Rules)
-  is probably much clearer.
+Rules are written in a Doman Specific Language (DSL). References for the rule syntax
+can be found in:
 
+* [Rule Syntax Guide](rules.html)
+* [Cloud Firewall Rules Reference](https://docs.joyent.com/public-cloud/network/firewall/cloud-firewall-rules-reference)
+* [CloudAPI Guide - Firewall Rules](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md#firewallrules)
 
 # Rules
 
@@ -473,3 +472,16 @@ something).
 ## 2015-03-23
 
 - Added `fields` option to the [ListRules](#ListRules) endpoint.
+
+## 2017-01-13
+
+- Migrated firewall rules from ufds to moray bucket to support the use
+  of certain special characters in tag names. (requires SAPI metadata
+  `FWRULE_VERSION` 3 or above)
+
+## 2018-03-16
+
+- Added `priority` support for firewall rules. (requires SAPI metadata
+  `FWRULE_VERSION` 4 or above, as well as firewaller and platform
+  image updates to include the changes in [TRITON-75](https://smartos.org/bugview/TRITON-75)
+  and [TRITON-637](https://smartos.org/bugview/TRITON-637))
-- 
2.21.0

